server/src/test/java/org/elasticsearch/index/mapper/IpScriptFieldTypeTests.java

```diff
@@ -228,9 +229,15 @@ protected Query randomTermQuery(MappedFieldType ft, SearchExecutionContext ctx)
     }
 
     @Override
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/106900")
     public void testTermsQuery() throws IOException {
-        try (Directory directory = newDirectory(); RandomIndexWriter iw = new RandomIndexWriter(random(), directory)) {
+        try (
+            Directory directory = newDirectory();
```

Comment:
Question - should we expand this fix to the individual term queries [here](https://github.com/elastic/elasticsearch/blob/3deb5ccfe50b413f0d327073f56277ac56642407/server/src/test/java/org/elasticsearch/index/mapper/IpScriptFieldTypeTests.java#L210)? Or are individual term queries not dependent on doc ordering?

Reply:
I think the same applies there too.